### PR TITLE
Gorger Rebalances & Visual Effects

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request

- Adds a brief immobilize to drain, making it slightly easier to land while moving at the cost of being more vulnerable when it first fires off

- Adds a beam vfx to Transfusion healing and Psylink's buff

- Rebalances Psylink slightly: Now, it takes half as long to channel, but collapses faster (at 50% gorger health instead of 20%)

- Buffs Oppose, reducing the price (from 100 to 80 )and improving the damage & stagger it does if you (somehow) fire it on top of a knocked down marine.
It had poor feedback with no visual effect, so I added a two part visual effect displaying the marine stagger and xeno heal ranges.
Also altered the desc to make it more clear how it already worked (Dmg for marine lying under you, stagger for adjacent marines, heal for nearby xenos)

Visual effects:

**Oppose**

https://github.com/user-attachments/assets/d6feeb21-4530-4383-9255-6f3d3098abdf
Shatter is marine stun range, particles are xeno heal range. 

**Transfusion**
https://github.com/user-attachments/assets/4657bcbe-0797-441b-a062-fec02f10b7b7

Psylink beam is identical to drone psylink, so I didn't grab any additional footage
## Why It's Good For The Game
Drain had a habit of instantly cancelling itself if the gorger in question was undergoing movement. This brief immobilize makes this much more difficult to immediately cancel, which should make it easier for most gorgers to pull off drains on moving targets.

Oppose is a slightly funky primordial ability, especially when compared to the explicitly caste-improving balancing of other primordials. It could use a buff so it's more "competitive" with base gorger skills in terms of blood use.
This pr reduces the relative cost somewhat (from costing 6 infusions worth of blood to 5) and makes it more rewarding to go for a slam on top of a marine, since it's pretty challenging to pull it off & rn there's very little incentive
Did you know oppose healed gorger? I didn't, it wasn't mentioned anywhere in code or descs, but it does. So I changed the desc to reflect how oppose actually works. 

I have other issues with Psylink(mainly that almost any situation where you'd use it would be better suited by just Going In yourself) but this should at least make it easier for players to use it effectively, while curbing the power slightly by making it fail more quickly.

It can be difficult to figure out how to use gorger properly (it has something of a rep as a niche for experienced players/noob trap) and the visual feedback for some abilities should make it slightly more clear how they work. 
Slight balance nudge & the drain fix should encourage more use of gorger and experimentation with it's abilities, since it's an interesting caste that deserves to be used more. 

(I may eventually have to nerf infusion, depending on how much this boosts gorger usage. We'll see how that turns out)

## Changelog
:cl:
add: the Gorger Psylink, Infusion, and Oppose abilities on Gorger now have vfx.
qol: gorger drain now briefly stops gorger, preventing them from instantly cancelling it on accident.
balance: buffs oppose, increasing the damage and stagger on a direct marine hit and reducing the cost slightly.
balance: gorger psylink now activates more quickly, but is cancelled more easily (at 50% gorger hp instead of 20%)
imageadd: A vfx shatter effect for Oppose.
spellcheck: removed repeated "Into the ground" from chat feedback on oppose.
/:cl:

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
does anyone know why 96x96.dmi lost size when I added something to it?

**Comment by Arturlang** [comment]
> does anyone know why 96x96.dmi lost size when I added something to it?

maybe because upstream had more data in it? See and check

**Comment by Sun-Soaked** [comment]
> > does anyone know why 96x96.dmi lost size when I added something to it?
> 
> maybe because upstream had more data in it? See and check

none of the upstream sprites are missing from this one. idk what's going on

**Comment by yyzsong** [comment]
this pr questions my judgement and thus I am not in favour of it.

(just a note for people who don't like to do math: this makes oppose require hitting 6 xenos to break even instead of 8)

**Review by Kuro020 — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm [hunk 1]
```diff
@@ -269,31 +283,45 @@
...
+			M.adjust_slowdown(3)
 			shake_camera(M, 3, 3)
 		else if(distance <= 1) //marines will only be staggerslowed if they're one tile away from you
 			shake_camera(M, 2, 2)
-			to_chat(M, span_highdanger("Blood swells up from the ground around you!"))
+			to_chat(M, span_highdanger("Blood shatters the ground around you!"))
 			M.adjust_stagger(2 SECONDS)
 			M.adjust_slowdown(3)
 
+/datum/action/ability/activable/xeno/oppose/proc/stop_particles(obj/effect/abstract/particle_holder/aoe_particles)
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm:322]: autodoc

**Review by Lumipharon — APPROVED** [review]
lgtm other than the autodoc

File: code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm (hunk 1 above)
Comment (Sun-Soaked) [reply code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm:323]: yea

**Comment by Sun-Soaked** [comment]
I did the ahh. autodoc btw

**Pull request merged**